### DISCUSSION TITLE
Padding connections (edges) when using the ELK layout engine

### DISCUSSION THREAD

**Discussion by emteelb** [terrastruct/d2#1410] [Q&A]
When using the ELK layout engine and the default "layered" algorithm, I have a few connections that are close together as shown:

![20230615-10:37:57_166x442_screengrab](https://github.com/terrastruct/d2/assets/106624800/b93b61c7-1f55-49e0-805c-c8f8f55cf45a)

Is it possible to specify a padding for connections? I think that they might be called "edges" in ELK-speak if I understand correctly from [here](https://www.eclipse.org/sirius/doc/specifier/diagrams/Diagrams.html):

> Edges are connections between a source and target element

**Comment by andrewferrier** [comment]
Potentially related issue: https://github.com/terrastruct/d2/issues/1221
